app/main.py

```diff
@@ -1,3 +1,61 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int or float) -> Distance:
```

Comment:
`or` is not used for annotations. Use `Union` or if you're using python 3.10 - `|` operator. And annotation itself is wrong. other can be Distance class too
Fix in other occurrences